pkg/env/kernel_package_upload.go

```diff
@@ -0,0 +1,6 @@
+package env
+
+var (
+	// DisableKernelPackageUpload is set to true to signal that kernel support package uploads should be disabled.
+	DisableKernelPackageUpload = RegisterBooleanSetting("ROX_DISABLE_KERNEL_PACKAGE_UPLOAD", false)
```

Comment:
Can we invert this setting to a positive indication like `ROX_ENABLE_KERNEL_PACKAGE_UPLOAD` to be easier to follow the code paths (and reducing inversions in if-cases)?